pulsar-common/src/test/java/org/apache/pulsar/common/api/DoubleByteBufTest.java

```diff
@@ -50,5 +50,22 @@ public void testReadableBytes() throws Exception {
             buf.skipBytes(64);
             assertEquals(buf.readableBytes(), 256 - 64 * (i + 1));
         }
+
+        buf.release();
+    }
+
+    @Test
+    public void testCapacity() throws Exception {
+
+        ByteBuf b1 = PooledByteBufAllocator.DEFAULT.heapBuffer(128, 128);
+        b1.writerIndex(b1.capacity());
+        ByteBuf b2 = PooledByteBufAllocator.DEFAULT.heapBuffer(128, 128);
+        b2.writerIndex(b2.capacity());
+        ByteBuf buf = DoubleByteBuf.get(b1, b2);
+
+        assertEquals(buf.capacity(), 256);
+        assertEquals(buf.maxCapacity(), 256);
+
+        buf.release();
```

Comment:
not necessary but as we don't have `release()` test in this class, should we also assert if b1 and b2 are also released and set as null by `DoubleByteBuf`

Reply:
Yes, I can add that check